spotifycli/spotifycli.py

```diff
@@ -49,6 +54,26 @@ def main():
         control_volume("-5%")
 
 
+def startShell():
+    while(True):
+        try:
+            command = input('Spotify > ')
+        except EOFError as exception:
+            print("Have a nice day!")
+            exit(0)
+
+        pid = os.fork()
+
+        if pid == 0:  # if executing context is child process
+            os.execlp('spotifycli', 'spotifycli', '--{}'.format(command))
+        elif pid > 0:
+            exit_status = os.waitpid(pid, 0)[1]  # wait for child to exit
+
+        else:  # error forking
+            print("System error encountered")
```

Comment:
We can remove comment and update error message to be more precise. E.g. `System error during process forking encountered`.